server/routes/hello.rb.sample

```diff
@@ -0,0 +1,10 @@
+# Rename this file to hello.rb to try it out
+
+class Routes::Hello < Bridgetown::Rack::Routes
+  route do |r|
+    # route: GET /hello/:name
+    r.get "hello", String do |name|
+      { hello: "friend #{name}" }
+    end
+  end
+end
```

Comment:
We don't need this, do we?

Reply:
Good catch, will remove the routes shenanigans

Reply:
Done